cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1611,6 +1650,31 @@ def test_create_groups(self):
         self.assertEqual(vertical_0.location, split_test.group_id_to_child['0'])
         self.assertEqual(vertical_1.location, split_test.group_id_to_child['1'])
 
+    def test_split_xblock_info_group_name(self):
```

Comment:
@mushtaqak I think we should move this test with utils test cases where we moved the logic.